jac/jaclang/langserve/engine.py

```diff
@@ -80,23 +80,28 @@ def update_modules(
                     impl_parent=self.modules[file_path],
                 )
 
-    def quick_check(self, file_path: str) -> bool:
+    def quick_check(self, file_path: str) -> tuple[bool, list[lspt.Diagnostic]]:
```

Comment:
Not a fan of returning a tuple here just for the purpose of test case checking. 

can you use the hover possition strategy for checking as in other tests?

Reply:
@kugesan1105 let me know if that works the `lsp.Position` type stuff in the hover tests.